webpack.config.js

```diff
@@ -0,0 +1,48 @@
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+    mode: 'development',
+    entry: './src/index.js',
+    output: {
+        path: path.resolve(__dirname,'dist'),
+        filename: 'bundle.js',
+        clean: true
+    },
+    devServer:{
+        static: './dist',
+    },
+    module:{
+        rules:[
+            {
+                test: /\.css$/i,
+                use: [MiniCssExtractPlugin.loader, "css-loader"],
```

Comment:
If you want to try module css this rule should be modified. (It's optional)

Reply:
Please remind where I can fide it in webpack docs